reference/release-notes.mdx

```diff
@@ -116,3 +116,34 @@ or (the non-shorthand version):
 ```
 [...] --secret key=value [...]
 ```
+
+# Breaking Change #2: Deprecation `path` and `port` for `liveness_probe`
+
+Starting from October of 2023, the usage of `path` and `port` for the `liveness_probe` will no longer be supported. Instead, the `command` option should be used for defining the liveness probe.
```

Comment:
@mtgoncalves1 are we planning on releasing this change in Oct?

Reply:
I'm not sure when should we release it. I just put `October` as a placeholder. What do you think?